_test/test_change_crystal/test_change_crystal_bragg_coarse.m

```diff
@@ -632,6 +634,36 @@ function test_rotmat_recovery(~)
 
             assertElementsAlmostEqual(corr.rotmat,rotmat_rec);
         end
+        function test_corrections_work_in_parallel(obj)
+            % testing if multifit works in parallel (actually no parallel 
+            % should be invoked) for this kind of task
```

Comment:
I do not understand this comment which contradicts itself.

Reply:
multifit should work regardless of parallel_multifit option is enabled or not. How it is achieved -- up to ticket Re #1724

Reply:
The comment still doesn't make any sense to me.

Reply:
modified